recipes-nodes/config-parser/config-parser.bb

```diff
@@ -10,8 +10,23 @@ SRC_URI = "file://config_parser.sh \
 
 S = "${WORKDIR}"
 
-# Default value for the config URL
-CLOUD_INIT_CONFIG_URL ?= "http://169.254.169.254/metadata/instance/compute/userData?api-version=2021-01-01&format=text"
+python () {
```

Comment:
Is this python script any different than the bb thing on the right?

Reply:
Yes, it is fetching it now from the bitbake caller env variable if passed prior to calling bitbake.
This way, we can pass this as build argument in the docker build as well similar to what we are doing with the searcher ssh key in the bob project